code/weapon/beam.h

```diff
@@ -133,6 +141,7 @@ typedef struct beam {
 	int		framecount;				// how many frames the beam has been active
 	int		flags;					// see BF_* defines 	
 	float		current_width_factor;		// applied to width due to shrink or grow
+	ubyte     shot_count;              // # of shots	
```

Comment:
Could this be moved to `beam_fire_info` instead? As part of the packet fixes PR I just send that struct to `send_beam_fired_packet()`, so if it's part of that it would be easier to use in that case.

Additionally, just make this an `int`. Data size is much less important than alignment in this case.

Reply:
Per the discussion on the multi packet fixes PR I was planning on simply reverting back to it being on `beam_info`. Did you want it somewhere else instead? It does have to persist with the beam in some way, or else look it up from `weapon_info` which seemed like it might be a bad idea.

Reply:
Having it back in `beam_info` is fine.